packages/docusaurus-utils/src/tags.ts

```diff
@@ -159,3 +214,82 @@ export function getTagVisibility<Item>({
     listedItems: items.filter((item) => !isUnlisted(item)),
   };
 }
+
+export function validateFrontMatterTags({
+  tags,
+  source,
+  options,
+}: {
+  tags: NormalizedTag[];
+  source: string;
+  options: Pick<PluginOptions, 'tagsFilePath' | 'onUnknownTags'>;
```

Comment:
Wonder how I should get the types from content plugin. (Also goes for `processFileTagsPath` options)
I tried to to something similar as `lastUpdateUtils` however I get a TS error that the type is unknown (just realised that lastUpdateUtils is also of type unknown)

<img width="456" alt="image" src="https://github.com/facebook/docusaurus/assets/29860391/0a849399-6b34-4488-829e-e7db99126721">